### PR TITLE
Update unbind.xml

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Updated signature as discussed in issue #384.

### CODE REVIEW

File: entries/unbind.xml [hunk 1]
```diff
@@ -23,9 +23,9 @@
   </signature>
   <signature>
     <added>1.0</added>
-    <argument name="event" type="Object">
-      <desc>A JavaScript event object as passed to an event handler.</desc>
-    </argument>
+    <argument name="event" type="Event">
+      <desc>A <a href="/category/events/event-object/"><code>jQuery.Event</code></a> object.</desc>
+    </argument>
```
Comment (arthurvr) [inline entries/unbind.xml]: Trailing whitespace :)

**Comment by AurelioDeRosa** [comment]
Sorry about that, thank you @arthurvr. I've remove the whitespace.

**Comment by arthurvr** [comment]
This PR LGTM, thanks @AurelioDeRosa! Anyone anything more to add, or do I merge it in?

**Comment by kswedberg** [comment]
LGTM. @arthurvr, feel free to merge. Thanks to both you and @AurelioDeRosa

**Comment by AurelioDeRosa** [comment]
Glad to help guys :) @arthurvr @kswedberg

**Comment by arthurvr** [comment]
Closed by https://github.com/jquery/api.jquery.com/commit/8642d2e4a6a54f25b54bf04998d6db108b1fee0e.

**Pull request closed without merging**